cmake/configure/configure_boost.cmake

```diff
@@ -23,6 +23,16 @@ SET(DEAL_II_WITH_BOOST ON # Always true. We need it :-]
 
 
 MACRO(FEATURE_BOOST_CONFIGURE_BUNDLED)
+
+  #
+  # Add rt to the link interface as well, boost/chrono needs it.
+  #
+  FIND_SYSTEM_LIBRARY(rt_LIBRARY NAMES rt)
+  MARK_AS_ADVANCED(rt_LIBRARY)
+  IF(NOT rt_LIBRARY MATCHES "-NOTFOUND")
+    SET(BOOST_LIBRARIES ${rt_LIBRARY})
```

Comment:
Out of curiosity, does this ensure that `-lrt` comes *after* the boost libraries on the command line?